chpl/chpl-service/src/main/java/gov/healthit/chpl/certifiedproduct/service/CertificationResultService.java

```diff
@@ -31,23 +36,28 @@
     private TestingFunctionalityManager testFunctionalityManager;
     private CertificationResultDetailsDAO certificationResultDetailsDAO;
     private SvapDAO svapDao;
+    private OptionalStandardDAO optionalStandardDAO;
+    private FF4j ff4j;
 
     @Autowired
     public CertificationResultService(CertificationResultRules certRules, CertificationResultManager certResultManager,
             TestingFunctionalityManager testFunctionalityManager, CertificationResultDetailsDAO certificationResultDetailsDAO,
-            SvapDAO svapDao) {
+            SvapDAO svapDAO, OptionalStandardDAO optionalStandardDAO, FF4j ff4j) {
         this.certRules = certRules;
         this.certResultManager = certResultManager;
         this.testFunctionalityManager = testFunctionalityManager;
         this.certificationResultDetailsDAO = certificationResultDetailsDAO;
-        this.svapDao = svapDao;
+        this.svapDao = svapDAO;
+        this.optionalStandardDAO = optionalStandardDAO;
+        this.ff4j = ff4j;
     }
 
     public List<CertificationResult> getCertificationResults(CertifiedProductSearchDetails searchDetails) throws EntityRetrievalException {
         List<SvapCriteriaMap> svapCriteriaMap = svapDao.getAllSvapCriteriaMap();
+        List<OptionalStandardCriteriaMap> optionalStandardCriteriaMap = optionalStandardDAO.getAllOptionalStandardCriteriaMap();
```

Comment:
Something in the method name that's called here reads like it should be plural... getAllOptionalStandardCriteriaMaps()?

Reply:
I can see that, but I was following the pattern of the SVAP map in the line above. I feel like changing one of them would imply I should change the other, and I don't know how far that might ripple